develop/tutorials/articles/300-osgi-basics/04-overriding-service-references.markdown

```diff
@@ -203,13 +203,16 @@ service references on the fly.
 
 1.  Create a configuration file named after the referencing component.
 
-    Here's the configuration file name for the example component:
+    Here's a configuration file name for the example component:
 
-        override.my.service.reference.OverrideMyServiceReference.config
+        override.my.service.reference.OverrideMyServiceReference.cfg
  
-    Note: For deploying to Liferay DXP DE 7.0 Fix Pack 8 or later, use file
-    suffix `.config`. For earlier Liferay DXP DE 7.0 Fix Packs or Liferay CE
-    Portal 7.0 GA3 or earlier, use suffix `.cfg`. 
+    Note: Liferay DXP DE 7.0 Fix Pack 8 and later and Liferay CE Portal 7.0 GA
+    and later support the Apache Felix ConfigAdmin implementation of OSGi
+    Configuration Admin files. Felix ConfigAdmin uses the file suffix `.config`
+    and supports additional types, such as arrays and vectors. The syntax for
+    `.cfg` and `.config` files can be found
+    [here](https://sling.apache.org/documentation/bundles/configuration-installer-factory.html). 
 
```

Comment:
@jhinkey 
Which GA version does this apply to?

Reply:
Ah, Liferay CE Poral 7.0 **GA4**

Good catch!

Reply:
Also, why is the example file a .cfg file? The note says, "Felix ConfigAdmin uses the file suffix .config."

Reply:
The .cfg syntax is easier to write. This example doesn't need the richness of the .config syntax. 

In any case, I force pushed another commit to prioritize showing .config files. Thanks.

Reply:
👍